contracts/governance/utils/Votes.sol

```diff
@@ -28,23 +28,16 @@ import "../../utils/cryptography/EIP712.sol";
  *
  * _Available since v4.5._
  */
-abstract contract Votes is Context, EIP712, IERC5805 {
+abstract contract Votes is Context, EIP712, Nonces, IERC5805 {
     using Checkpoints for Checkpoints.Trace224;
-    using Counters for Counters.Counter;
 
     bytes32 private constant _DELEGATION_TYPEHASH =
         keccak256("Delegation(address delegatee,uint256 nonce,uint256 expiry)");
 
     mapping(address => address) private _delegation;
-
-    /// @custom:oz-retyped-from mapping(address => Checkpoints.History)
```

Comment:
As far as I can tell this was not due to conflict resolution. I'm guessing you're removing these annotations because they're no longer needed for 5.0?

I would've preferred to have a clean separation between what is merge conflict resolution and what is further 5.0 work. I wouldn't go as far as to say we should revert this but let's please not add any more "new" things in this PR.